.github/workflows/postbench.yml

```diff
@@ -0,0 +1,61 @@
+name: PostBenchmarks
+on:
+  workflow_run:
+    workflows: [RunUploadBenchmarks]
+    types: [completed]
+    branches: [master, benchx]
+jobs:
```

Comment:
can we give this job granular permissions?

Reply:
I expect that to get `permissions: write-all` automatically, but I can add it explicitly to raise our chances we get it right in the first trial.

Reply:
by more granular I was actually thinking we restrict the write permissions to the ones we need (probably just PR)

Reply:
ah okey, you mean something like that 
```
permissions:
  pull-requests: write
```
I am not sure if that's enough, however since there are many permissions [here ](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#defining-access-for-the-github_token-scopes). Can we push like that and later narrow down the permissions ? Similar to the [previous discussion](https://github.com/JuliaGraphs/Graphs.jl/pull/350#discussion_r1519502292) let's keep it simple in the beginning because debugging is hard in these circumstances.

Reply:
okay for me